[2025-03-12T15:21:00.073+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T15:21:00.098+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.end_pipeline manual__2025-03-12T15:20:57.288293+00:00 [queued]>
[2025-03-12T15:21:00.107+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.end_pipeline manual__2025-03-12T15:20:57.288293+00:00 [queued]>
[2025-03-12T15:21:00.108+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T15:21:00.129+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): end_pipeline> on 2025-03-12 15:20:57.288293+00:00
[2025-03-12T15:21:00.135+0000] {standard_task_runner.py:72} INFO - Started process 258 to run task
[2025-03-12T15:21:00.145+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'end_pipeline', 'manual__2025-03-12T15:20:57.288293+00:00', '--job-id', '170', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp8xkbqapd']
[2025-03-12T15:21:00.151+0000] {standard_task_runner.py:105} INFO - Job 170: Subtask end_pipeline
[2025-03-12T15:21:00.192+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T15:21:00.252+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.end_pipeline manual__2025-03-12T15:20:57.288293+00:00 [running]> on host ***-scheduler
[2025-03-12T15:21:00.419+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='end_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T15:20:57.288293+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T15:20:57.288293+00:00'
[2025-03-12T15:21:00.421+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T15:21:00.437+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-12T15:21:00.438+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Pipeline terminé"']
[2025-03-12T15:21:00.451+0000] {subprocess.py:99} INFO - Output:
[2025-03-12T15:21:00.453+0000] {subprocess.py:106} INFO - Pipeline terminé
[2025-03-12T15:21:00.453+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-12T15:21:00.487+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T15:21:00.488+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=end_pipeline, run_id=manual__2025-03-12T15:20:57.288293+00:00, execution_date=20250312T152057, start_date=20250312T152100, end_date=20250312T152100
[2025-03-12T15:21:00.562+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T15:21:00.596+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-12T15:21:00.601+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
